lib/modules/manager/nix/artifacts.spec.ts

```diff
@@ -36,17 +36,14 @@ process.env.CONTAINERBASE = 'true';
 const config: UpdateArtifactsConfig = {};
 const lockMaintenanceConfig = { ...config, isLockFileMaintenance: true };
 const updateInputCmd = `nix \
-    --extra-experimental-features nix-command \
-    --extra-experimental-features flakes \
+--extra-experimental-features 'nix-command flakes' \
```

Comment:
please do this argument refactor in a separate PR to be merged first, so we've less changes here

Reply:
#32289